src/schema/mutation/editApiConfiguration.mutation.ts

```diff
@@ -41,42 +42,33 @@ export default {
         );
       }
       const ability: AppAbility = user.ability;
-      if (
-        !args.name &&
-        !args.status &&
-        !args.authType &&
-        !args.endpoint &&
-        !args.pingUrl &&
-        !args.graphQLEndpoint &&
-        !args.settings &&
-        !args.permissions
-      ) {
+
+      // Check if any of required arguments for a valid update are provided.
+      // Else, send error
+      // cloneDeep coupled with omit allows to filter out the 'id' field
+      if (isEmpty(cloneDeep(omit(args, ['id'])))) {
         throw new GraphQLError(
           context.i18next.t(
             'mutations.apiConfiguration.edit.errors.invalidArguments'
           )
         );
       }
-      const update = {};
+      // See if API name is usable
       if (args.name) {
         validateApi(args.name);
       }
-      Object.assign(
-        update,
-        args.name && { name: args.name },
-        args.status && { status: args.status },
-        args.authType && { authType: args.authType },
-        args.endpoint && { endpoint: args.endpoint },
-        args.graphQLEndpoint && { graphQLEndpoint: args.graphQLEndpoint },
-        args.pingUrl && { pingUrl: args.pingUrl },
-        args.settings && {
+      // Create the update document
+      const update = {
```

Comment:
@RenzoPrats 
ℹ️  Same idea there, shorter code for same purpose

Reply:
the idea is really to reduce the number of lines of code when possible, and lodash is a great tool for that 👍